src/clojure/duratom/core.clj

```diff
@@ -261,6 +261,21 @@
                                            (:write rw))}
                    (select-keys rw [:commit-mode])))))
 
+(def default-redis-rw
+  {:read  identity ;Redis library Carmine automatically uses Nippu for serialization/deserialization
+   :write identity ;Redis library Carmine automatically uses Nippu for serialization/deserialization
+   :commit-mode DEFAULT_COMMIT_MODE}) ;; technically not needed but leaving it for transparency)
+
+(defn redis-atom [db-config key-name lock initial-value rw]
```

Comment:
Could you make this a multi-arity fn, similar to `postgres-atom`?

Reply:
Sure. Done.